vendor/phantomjs/render.js

```diff
@@ -55,7 +55,8 @@
 
         var rootScope = body.injector().get('$rootScope');
         if (!rootScope) {return false;}
-        return rootScope.panelsRendered;
+        var panelsToLoad = window.angular.element('div.panel').length;
```

Comment:
Without checking window.angular.element('div.panel-drop-zone').length in case of rendering full dashboard with all panels, render.js ends only on 'tries=1000' count, with my code it returns faster than 'tries' count exceed limit.